api/server.js

```diff
@@ -15,13 +19,13 @@ const db = require("./app/models");
 
 //force can be used in a development environment but not in production as it will drop the table first and create again, alter will match the existing model and change the table accordingly. Alter is also not advisable in production as will delete data of the columns removed or type changed, but if you want to avoid migrations and update your table this is the option.
 
-db.sequelize.sync()
-  .then(() => {
-    console.log("All models were synchronised successfully.");
-  })
-  .catch((err) => {
-    console.log("Failed to sync db: " + err.message);
-  });
+//db.sequelize.sync()
+//  .then(() => {
+//    console.log("All models were synchronised successfully.");
+//  })
+//  .catch((err) => {
```

Comment:
Should we just delete rather than comment out?

Reply:
I left that out as it was useful when syncing in cases schema design changes. but will delete and refer to earlier commits instead